src/BasicBehaveEngine/nodes/variable/VariableGet.ts

```diff
@@ -20,7 +20,12 @@ export class VariableGet extends BehaveEngineNode {
         this.graphEngine.processNodeStarted(this);
 
         const result: Record<string, IValue> = {};
-        result[this.variables[this._variable].id] = this.variables[this._variable];
+        const res = this.variables[this._variable];
+        // TODO It's unclear when the value needs to be a glTF value (always an array)
```

Comment:
the spec says all returned values should be arrays https://github.com/KhronosGroup/glTF/blob/interactivity/extensions/2.0/Khronos/KHR_interactivity/Specification.adoc#44-variables single value types like int, bool, float should be singleton arrays